src/components/button/button.tsx

```diff
@@ -32,9 +32,12 @@ export const Button: FC<Props> = ({
     activeColorConfig,
     disabledColorConfig = activeColorConfig
   } = styleConfig;
-  const { titleColor, iconColor = titleColor, backgroundColor, borderColor = backgroundColor } = disabled
-    ? disabledColorConfig
-    : activeColorConfig;
+  const {
+    titleColor,
+    iconColor = titleColor,
+    backgroundColor,
+    borderColor = backgroundColor
+  } = disabled ? disabledColorConfig : activeColorConfig;
```

Comment:
What value do you have here?
![Uploading image.png…]()

Reply:
120